static/app/components/events/contexts/index.tsx

```diff
@@ -35,12 +35,13 @@ export interface ContextItem {
 
 export function getOrderedContextItems(event): ContextItem[] {
   const {user, contexts} = event;
+  const {data: customUserData, ...userContext} = user;
 
   const {feedback, response, ...otherContexts} = contexts ?? {};
   const orderedContext: [ContextItem['alias'], ContextValue][] = [
     ['response', response],
     ['feedback', feedback],
-    ['user', user],
+    ['user', {...userContext, ...customUserData}],
```

Comment:
Custom properties are storied on the user object under the 'data' field. We do this to flatten it and remove the `data` key.